source/androidx.viewpager2/viewpager2/transforms/Metadata.xml

```diff
@@ -0,0 +1,18 @@
+﻿<metadata>
+    <attr
+        path="/api/package[@name='androidx.viewpager2.adapter']/class[@name='FragmentStateAdapter']/method[@name='onCreateViewHolder' and count(parameter)=2 and parameter[1][@type='android.view.ViewGroup'] and parameter[2][@type='int']]"
+        name="final"
+        >
+        false
```

Comment:
Is this making the method virtual again? Never done this before so it would be interesting to see the use case.

Reply:
Simply to work around `cannot override inherited member`